### PR TITLE
AUTH-352: Place common github actions into repo

### PR DESCRIPTION
**Pull request by chazdnato**
As an assist to https://github.com/ecobee/auth0-mfa-count/pull/13, we will be hosting all our self-created actions in a single public repo. This is the start of said repo.

### CODE REVIEW

**Review by jessicawoo — APPROVED** [review]

File: push_docker_gcr/action.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: 'GCP: Creaet and push docker'
```
Comment (jessicawoo) [inline push_docker_gcr/action.yml]: s/Creaet/Create

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+version: 2
+updates:
+
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
```
Comment (jessicawoo) [inline .github/dependabot.yml:5]: So we're going to upgrade all dependencies for Github Actions, not just security-related ones?

File: push_docker_gcr/action.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+description: 'Creates docker, pushes to gcr.io'
```
Comment (jessicawoo) [inline push_docker_gcr/action.yml]: Creates docker image?

File: push_docker_gcr/action.yml (hunk 1 above)
Comment (chazdnato) [reply push_docker_gcr/action.yml]: fixed

File: .github/dependabot.yml (hunk 2 above)
Comment (chazdnato) [reply .github/dependabot.yml:5]: security related are at the org level, this whole file was as per security team recommendation (as it stands, there is nothing to do, as the repo is just yaml)

File: push_docker_gcr/action.yml (hunk 3 above)
Comment (chazdnato) [reply push_docker_gcr/action.yml]: fixed

**Pull request merged**